data/registry/instrumentation-go-splunkdns.yml

```
@@ -13,3 +13,4 @@ authors:
 urls:
   repo: https://github.com/signalfx/splunk-otel-go/tree/main/instrumentation/github.com/miekg/dns/splunkdns
 createdAt: 2022-01-20
+isFirstParty: true
```

```suggestion
isFirstParty: false
```

this is not a first party instrumentation